src/libs/GrowlNotification.js

```diff
@@ -0,0 +1,101 @@
+import React, {
+    forwardRef, useImperativeHandle, useRef, useState,
+} from 'react';
+import {
+    Text, View, Animated, Platform,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../styles/colors';
+import Icon from '../components/Icon';
+import {Checkmark, Exclamation} from '../components/Icon/Expensicons';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../components/withWindowDimensions';
+
+const types = {
+    success: {
+        icon: Checkmark,
+        iconColor: colors.green,
+    },
+    error: {
+        icon: Exclamation,
+        iconColor: colors.red,
+    },
+    warning: {
+        icon: Exclamation,
+        iconColor: colors.yellow,
+    },
+};
+
+const defaultOptions = {
+    bodyText: '',
+    type: 'success',
+};
+
+const outDistance = -255;
+
+const GrowlNotification = forwardRef(({isSmallScreenWidth}, ref) => {
+    const slideDown = useRef(new Animated.Value(outDistance)).current;
+    const [options, setOptions] = useState(defaultOptions);
```

Comment:
Also please carefully read our [JavaScript coding standards for this repo](https://github.com/Expensify/Expensify.cash/blob/main/STYLE.md). You'll notice that we don't use react hooks anywhere in this repo.

Reply:
My bad. I've refactored the code now

Reply:
No worries, it's your first PR in this repo 😄! I'll give this PR another review now 👍